src/project-overlay/ProjectOverlay.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  Modal,
+  Button,
+  Grid,
+  Text,
+  Flex,
+  Image,
+  ActionIcon,
+  Avatar,
+} from "@mantine/core";
+import "./ProjectOverlay.css";
+
+type TeamMember = {
+  name: string;
+  role: string;
+  imageSrc: string;
+};
+
+type ProjectOverlayProps = {
+  name: string;
+  years: string[];
+  projectDescription: string;
+  nonprofitDescription: string;
+  imageSrc: string;
+  team: TeamMember[];
+  webLink: string;
+  githubLink: string;
+  isOpened: boolean;
+  closeModal: () => void;
+};
+
+export default function ProjectOverlay(props: ProjectOverlayProps) {
+  return (
+    <>
+      <Modal
+        opened={props.isOpened}
+        onClose={props.closeModal}
+        size={"xl"}
+        centered
+        withCloseButton={false}
+        radius={15}
+      >
+        <Grid gutter={"sm"}>
+          <Grid.Col span={4}>
+            <Image radius="md" h={"100%"} src={props.imageSrc} />
+          </Grid.Col>
+          <Grid.Col span="auto" className="modal-content">
+            <Modal.Header className="header">
+              <Flex direction={"column"}>
+                <Modal.Title className="title">{props.name}</Modal.Title>
+                <Text className="years">{props.years.join(", ")}</Text>
+              </Flex>
+              <Flex gap="xs">
+                <ActionIcon
+                  component="a"
+                  size={45}
+                  href={""}
+                  variant="transparent"
+                >
+                  <Image src="/globe-fill.svg" alt="Web Icon" />
+                </ActionIcon>
+                <ActionIcon
+                  component="a"
+                  size={45}
+                  href={""}
+                  variant="transparent"
+                >
+                  <Image src="/mark-github-24.svg" alt="GitHub Icon" />
+                </ActionIcon>
+              </Flex>
+            </Modal.Header>
+            <Modal.Body style={{ gap: "16px" }}>
+              <Flex direction={"column"}>
+                <Text className="section-header">DESCRIPTION OF PROJECT</Text>
+                <Text className="section-description">
+                  {props.projectDescription}
+                </Text>
+              </Flex>
+              <Flex direction={"column"}>
+                <Text className="section-header">ABOUT NON-PROFIT</Text>
+                <Text className="section-description">
+                  {props.nonprofitDescription}
+                </Text>
+              </Flex>
+              <Flex direction={"column"}>
+                <Text className="section-header">TEAM</Text>
+                {TeamsGrid(props.team)}
+              </Flex>
+            </Modal.Body>
+          </Grid.Col>
+        </Grid>
+      </Modal>
+    </>
+  );
+}
+
+function TeamsGrid(team: TeamMember[]) {
+  const teamMembers = team.map((member: TeamMember) => (
+    <Grid.Col span={{ base: 10, md: 6, lg: 6 }} key={member.name}>
```

Comment:
Breakpoints are causing TeamMember names to get cut off around (450px to 600px) and (400px to 320px). Maybe change to `span={{base: 15, sm: 10, md: 6, lg: 6}}` ?


<details>
  <summary>Image</summary>
  
![image](https://github.com/uoftblueprint/website-v2/assets/81433191/9a9b1556-c6e9-42cd-966a-d9456b50f710)</details>